JSON/Simon Forgets.json

```diff
@@ -0,0 +1,46 @@
+{
+  "Author": "Kusane, noi2coco",
+  "Compatibility": "Untested",

```

Comment:
untested is not usually used as a compatiblity, since it's assumed you do a base level of testing when you create the module